datadog_checks_base/datadog_checks/base/__about__.py

```diff
@@ -1,4 +1,4 @@
 # (C) Datadog, Inc. 2018-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-__version__ = "15.1.0"
+__version__ = "15.2.0"
```

Comment:
Was this intended? We should keep the release bump in a separate PR that can be pushed after this one gets merged. See our release process here: https://datadoghq.dev/integrations-core/process/integration-release/

So let's consider reverting the release commits from this branch? :)

Reply:
Ah ok, I misunderstood the comment here: https://github.com/DataDog/integrations-core/pull/7837#pullrequestreview-515438570. Reverted!